src/sklearn_evaluation/plot/clustering.py

```diff
@@ -141,3 +143,168 @@ def _clone_and_score_clusterer(clf, X, n_clusters):
     clf = clone(clf)
     setattr(clf, 'n_clusters', n_clusters)
     return clf.fit(X).score(X), time.time() - start
+
+
+def silhouette_plot(X,
```

Comment:
We need to track the usage of `def silhouette_plot` and `def silhouette_plot_from_results`. We can do it with `@SKLearnEvaluationLogger.log(feature='plot')`

Reply:
added